### ISSUE TITLE
Error when pressing the Read Mode button

### ISSUE THREAD

**Issue by Mastade** [fork-maintainers/iceraven-browser#717]
The last 3-4 browser updates have added a problem to me, when clicking on the Read Mode button (near the url address), an error often appears - 
Cannot Complete Request
Additional information about this problem or error is currently unavailable.
Try Again 
I have completely cleared all the application data, but the problem remains. Miui 13, Android 12 phone(stock firmware), please help me solve the problem
I am attaching a screenshot with an error.
![photo1703073825](https://github.com/fork-maintainers/iceraven-browser/assets/91428739/131f8508-e506-4868-9652-c622abd8cfc3)

**Comment by lividhen** [comment]
Are there specific websites where this occurs? I have not been able to trigger it as of yet.

**Comment by Mastade** [comment]
No, such a problem can occur on any site, but not always, but every 3-5 times

**Comment by lividhen** [comment]
Go to the settings of iceraven, scroll down to the bottom and hit about, tap the iceraven logo 5-7 times to enable the debug menu. Back out of the about section and scroll to the bottom where you will see an option called start profiler. Select that and hit the network option and hit start profiler. Back out of settings and trigger the error. Then go back into settings and scroll down to profiler again and tap it to stop it. It will upload the profile and copy a link to it. Paste that link here. Idk if it will show anything useful but my guess would be it is some network issue, so maybe we can find something.

**Comment by Mastade** [comment]
Thank you for wanting to help! I did everything according to your instructions.
https://profiler.firefox.com/public/e7zxp6q0apd4p1dh1s8mavwgzy9xgjytybgsq5g

Plus I copied the URL during the error

moz-extension://84430e0c-6481-46fe-b734-ed6063cbc555/readerview.html?url=https%3A%2F%2Fru.m.wikipedia.org%2Fwiki%2F%25D0%25A0%25D1%2583%25D1%2581%25D1%2581%25D0%25BA%25D0%25B0%25D1%258F_%25D0%25B4%25D0%25BE%25D1%2580%25D0%25B5%25D1%2584%25D0%25BE%25D1%2580%25D0%25BC%25D0%25B5%25D0%25BD%25D0%25BD%25D0%25B0%25D1%258F_%25D0%25BE%25D1%2580%25D1%2584%25D0%25BE%25D0%25B3%25D1%2580%25D0%25B0%25D1%2584%25D0%25B8%25D1%258F&id=b256da7c-a3ae-4bee-b8c3-4c90957a33f1&colorScheme=dark

My smartphone Poco X3 pro - miui 13.0.4, android 12

**Comment by akliuxingyuan** [comment]
upstream issue https://bugzilla.mozilla.org/show_bug.cgi?id=1863662

**Comment by Mastade** [comment]
Thanks a lot for the link, as far as I understand, the error is in firefox itself and so far it cannot be fixed, I just need to wait for the fix in new versions

**Comment by akliuxingyuan** [comment]
reader mode is gone.
